things/api/things/http/responses.go

```diff
@@ -52,7 +55,7 @@ func (res thingRes) Code() int {
 func (res thingRes) Headers() map[string]string {
 	if res.created {
 		return map[string]string{
-			"Location": fmt.Sprintf("/things/%s", res.id),
+			"Location": fmt.Sprintf("/things/%s", res.ID),
```

Comment:
There is no point in `Location` header if you're going to return `id` as part of JSON response. Think about splitting responses for creating a single thing, and multiple things.

Reply:
@anovakovic01 I left this in place to keep the current behavior for a single thing creation.  Right now it will return a `thingRes` with the `id` in the`Location` header without a JSON response.  Multiple thing creation will return a `thingsRes` with an empty `Location` header and a JSON response.